thoth/package_extract/image.py

```diff
@@ -477,17 +489,14 @@ def _get_python_interpreters(path: str) -> dict:
             if len(parts) == 2 and parts[0] == "Python":
                 version_ = line.rstrip()
         except Exception as exc:
-            _LOGGER.warning(
-                "Failed to get python version: %s", str(exc))
+            pass
```

Comment:
Please do not silently pass exceptions - exceptions can help with debugging and information they produce can be informative. You can log a warning here with the exception message.

Reply:
Thanks, added a warning.